airgun/entities/usergroup.py

```diff
@@ -0,0 +1,64 @@
+from navmazing import NavigateToSibling
+
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep, navigator
+from airgun.views.usergroup import UserGroupCreateView, UserGroupDetailsView, UserGroupsView
+
+
+class UserGroupEntity(BaseEntity):
+
+    def create(self, values):
+        view = self.navigate_to(self, 'New')
+        view.fill(values)
+        self.browser.selenium.implicitly_wait(3)
```

Comment:
not a chance that we use implicit waits in our code, please find alternative solutions (e.g. wait_for library or any other visibility validations)

Reply:
Oh, that's embarrassing. I have used it during debugging sessions and never meant to actually commit it, but forgot to delete it. Good catch!